big-game/src/app/competition-page/competitions/competitions.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Breed } from 'src/app/breed';
+import { BreedService } from 'src/app/services/breed.service';
+import { CompetitionService } from 'src/app/services/competition.service';
+
+@Component({
+	selector: 'app-competitions',
+	templateUrl: './competitions.component.html',
+	styleUrls: [ './competitions.component.css' ]
+})
+export class CompetitionsComponent implements OnInit {
+	allBreeds: Breed[];
+	breedSelected: string;
+	compTypes = ['Trotting','Barrel Racing', 'Reining', 'Cutting', 'Trail Class', 'Western Pleasure'];
```

Comment:
hardcoding

Reply:
done by design - there is no db or intention for one at this point in the game for  types of competitions.  this is only to populate a drop-down with options